etl/steps/data/garden/energy/2022-08-03/electricity_mix.meta.yml

```diff
@@ -69,13 +69,13 @@ dataset:
       date_accessed: 8th July 2022
       url: https://www.bp.com/en/global/corporate/energy-economics/statistical-review-of-world-energy.html
     -
-      name: Our World in Data based on Ember's Global Electricity Review (2022).
+      name: Our World in Data based on Ember's Global Electricity Review (2022)
       published_by: Ember
       publication_year: 2022
       date_accessed: 2022-08-01
       url: https://ember-climate.org/data/data-catalogue/yearly-electricity-data/
     -
-      name: Our World in Data based on Ember's European Electricity Review (2022).
+      name: Our World in Data based on Ember's European Electricity Review (2022)
```

Comment:
Minor thing, but probably this does not belong in this PR?

Reply:
I think we're doing worse things than this :D.

Reply:
The thing is that I randomly picked this file to use as a test for the main changes of this PR :-).